mikeio/custom_exceptions.py

```diff
@@ -0,0 +1,10 @@
+class FileDoesNotExist(Exception):
```

Comment:
Seems redundant, better to use [FileNotFoundError](https://docs.python.org/3/library/exceptions.html#FileNotFoundError)

Reply:
Right. So I will change to FileNotFoundError in that case 👍